packs/src/races/dragonborn.json

```diff
@@ -1,85 +1,184 @@
       "chat": "",
       "unidentified": ""
     },
-    "source": "SRD 5.1",
-    "activation": {
-      "type": "",
-      "cost": null,
-      "condition": ""
-    },
-    "duration": {
-      "value": "",
-      "units": ""
-    },
-    "cover": null,
-    "target": {
-      "value": null,
-      "width": null,
-      "units": "",
-      "type": ""
-    },
-    "range": {
-      "value": null,
-      "long": null,
-      "units": ""
-    },
-    "uses": {
-      "value": null,
-      "max": "",
-      "per": null,
-      "recovery": ""
-    },
-    "consume": {
-      "type": "",
-      "target": null,
-      "amount": null
-    },
-    "ability": null,
-    "actionType": "",
-    "attackBonus": "",
-    "chatFlavor": "",
-    "critical": {
-      "threshold": null,
-      "damage": ""
+    "source": {
+      "custom": "SRD 5.1",
+      "book": "",
+      "page": "",
+      "uuid": "",
+      "license": ""
     },
-    "damage": {
-      "parts": [],
-      "versatile": ""
+    "identifier": "dragonborn",
+    "advancement": [
+      {
+        "_id": "QIrBmvSmNOTEJLWn",
+        "type": "ItemChoice",
+        "configuration": {
+          "hint": "You have draconic ancestry. Choose one type of dragon from the list. Your breath weapon and damage resistance are determined by the dragon type.",
```

Comment:
Should we add a trait advancement that offers a choice of 1 damage resistance also?